lib/wanda_web/controllers/health_controller.ex

```diff
@@ -10,12 +10,15 @@ defmodule WandaWeb.HealthController do
   }
 
   operation :ready,
-    summary: "Wanda ready",
-    tags: ["Platform"],
-    description: "Check if Wanda is ready",
+    summary: "Wanda ready.",
+    tags: ["Wanda Platform"],
```

Comment:
Could we leave this as `Platform`. Adding `Wanda` looks redundant

Reply:
I partially agree with it. The problem with `Platform` is that it overlaps with the same tag in TrentoWeb. As we might need to combine both specs in a single file (for the MCP), this name is not ok. On the other hand, adding `Wanda` is indeed redundant. 
What do you think of something like `Checks Platform` ?